markdown/articles/2020/03/22/DDFAQs.md

```diff
@@ -0,0 +1,600 @@
+solidly based in {{UB}} however the rest of the available colours are all useable within the
+shell. Below will be a summary of each variant and the colour options they tend to run
+followed by some in depth discussion as to what the different colour splashes bring to the table
+in terms of tools.
+
+**DDFT**
+Normally base {{UBR}} with the option to splash {{W}} or {{G}}.
+
+**DDEFT**
+Normally base {{UBR}}, unlikley to run splash colours.
+
+**Meandeck**
+Normally {{UBG}} or {{UBW}} with the option to splash {{W}}, {{R}} or {{G}}.
+
+
+
+### 3.1 White
+
+White is almost exclusively played in Meandeck lists however you could easily incorporate
+it into DDFT. Maindeck, {{W}} provides protection spells mostly in the form of {{Orim's
+Chant}}, {{Silence}} or {{Teferi, Time Raveler}} the last of which can also act as a 
+method to draw into a Doomsday pile or a mini card advantage engine when the '-1' is
+coupled with {{Arcum's Astrolabe}}. {{Abeyance}} is another great option that also cantrips
+and so makes a good initiation spell for drawing into a Doomsday pile.
+
+From the sideboard {{W}} offers a plan B in the form of {{Monastery Mentor}} that plays
+well with the cantrip dense lists that are run. Removal comes in the form of {{Disenchant}}
+or {{Wear//Tear}} for artifact or enchantment hate and {{Swords to Plowshares}} to deal
+with problematic creatures. You also gain some additional graveyard hate options in the
+form of {{Rest in Peace}}.
+
+Teferi and Mentor are probably the biggest boons for playing {{W}}.
+
+
+### 3.2 Red
+
+Red is obviously a staple for DDEFT and often for DDFT too. It enables {{Burning Wish}} in
+the main deck as a tutor for both the combo elements and other toolbox cards. From the side
+it can provide access to alternative win conditions like {{Empty the Warrens}} and a 
+plethora of removal for different scenarios such as {{Abrade}}, {{By Force}} or 
+{{Rending Volley}} to name a few. {{R}} also gives you access to {{Pyroblast}}/
+{{Red Elemental Blast}} effects which are incredibly powerful tools against any {{U}}
+based matchup.
+
+Empty the Warrens and Red Blasts are the biggest boons for playing {{R}}.
+
+
+### 3.3 Green
+
+Green provides a slightly cheaper package in terms of mana cost. Maindeck it provides 
+access to protection in the form of {{Veil of Summer}} and super efficient removal
+in the form of {{Abrupt Decay}}. From the sideboard additional Veils/Decays may be
+brought in alongside other elements such as  {{Carpet of Flowers}} or {{Xantid Swarms}}.
+{{G}} does not have a plan B associated with it however some have explored the use of 
+{{Oko, Thief of Crowns}} as a potential option or {{Uro, Titan of Nature's Wrath}}.
+
+Veil and Decay are the biggest boons for playing {{G}}.
+
+
+### 3.4 No Splash
+
+There is some potential for running a straight {{UB}} list with no splash colours. For 
+a speed focussed list, use of {{Wishclaw Talisman}} and {{Echo of Eons}} could
```

Comment:
"focused" has 1 s, unless the British are even crazier than I thought

Reply:
Corrected.